mava/wrappers/jumanji.py

```diff
@@ -208,30 +208,44 @@ def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
 
         # TARGET = 3 = The number of different types of items on the grid.
         def create_agents_view(grid: chex.Array) -> chex.Array:
-            positions = jnp.where(grid % TARGET == POSITION, True, False)
-            targets = jnp.where((grid % TARGET == 0) & (grid != EMPTY), True, False)
-            paths = jnp.where(grid % TARGET == PATH, True, False)
-            position_per_agent = jnp.where(grid == POSITION, True, False)
-            target_per_agent = jnp.where(grid == TARGET, True, False)
+            positions = jnp.where(grid % TARGET == POSITION, jnp.ceil(grid / TARGET), 0)
+            targets = jnp.where((grid % TARGET == 0) & (grid != EMPTY), jnp.ceil(grid / TARGET), 0)
+            paths = jnp.where(grid % TARGET == PATH, 1, 0)
+            position_per_agent = jnp.where(grid == POSITION, 1, 0)
+            target_per_agent = jnp.where(grid == TARGET, 1, 0)
             agents_view = jnp.stack(
                 (positions, targets, paths, position_per_agent, target_per_agent), -1
             )
             return agents_view
 
+        def aggregate_rewards(
+            timestep: TimeStep,
+        ) -> TimeStep[Observation]:
+            """Aggregate individual rewards and discounts across agents."""
+            team_reward = jnp.sum(timestep.reward)
+            reward = jnp.repeat(team_reward, self.num_agents)
+            discount = jnp.repeat(jnp.max(timestep.discount), self.num_agents)
```

Comment:
I still think we should leave the discounts as per agent, especially since we don't even use them in PPO.